.github/workflows/build-test.yml

```diff
@@ -268,8 +268,10 @@ jobs:
 
       # Nightly is only needed for the `--profile bench` line. Once that is stabilized,
       # then this line is no longer needed.
-      - name: Install nightly toolchain
-        run: rustup toolchain install nightly
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2020-04-04
```

Comment:
Why are you picking such old versions of nightly?  Seems like you should pick the most recent version that works (something in 2021, not 2020)